Dockerfile broken - Cannot find module /usr/local/src/app/plugins.json

Prerequisites
* [x] Are you running the latest version?
* [x] Are you able to consistently reproduce the issue?
* [x] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)

## Issue Description
On a fresh clone of `reaction-development-platform`, after running `make`, the `reaction` container fails to start with the following error:

```
ERROR Reaction: Cannot find module /usr/local/src/app/plugins.json imported from /usr/local/src/app/node_modules/@reactioncommerce/api-core/src/importPluginsJSONFile.js
  Error: Cannot find module /usr/local/src/app/plugins.json imported from /usr/local/src/app/node_modules/@reactioncommerce/api-core/src/importPluginsJSONFile.js
      at Loader.resolve [as _resolve] (internal/modules/esm/default_resolve.js:84:13)
      at Loader.resolve (internal/modules/esm/loader.js:73:33)
      at Loader.getModuleJob (internal/modules/esm/loader.js:147:40)
      at Loader.import (internal/modules/esm/loader.js:131:28)
      at importModuleDynamically (internal/modules/esm/translators.js:75:23)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! reaction-api@3.7.0 start: `node --experimental-modules --experimental-json-modules ./src/index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the reaction-api@3.7.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

The same issue happens when running `docker build .`. However, the issue does not happen when running `reaction` in development mode `make dev-link-reaction`. This lead me to believe that the Dockerfile needs some kind of an update to work with the new `plugins.json` file.

## Steps to Reproduce
1. Clone a fresh new `reaction-development-platform` (making sure to `make clean` any previous platform to wipe volumes)
2. Run `make`
3. Print logs for `reaction_api_1` and notice the error being thrown.
4. Run `docker build .` and notice the same error.
5. Run `make dev-link-reaction && make start-reaction` and notice `reaction` starting up fine.

## Possible Solution
Tweak something in the Dockerfile to make the `plugins.json` file work. Maybe it's not being copied into the container?


### Versions
3.7.0

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,3 +9,4 @@ RUN cd node_modules/sharp && (node install/libvips && node install/dll-copy && p
 
 # The base image copies /src but we need to copy additional folders in this project
 COPY --chown=node:node ./public ./public
+COPY --chown=node:node ./plugins.json ./plugins.json

